src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/ServerAddressesFeature.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Hosting.Server.Features;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal
+{
+    internal class ServerAddressesFeature : IServerAddressesFeature
```

Comment:
@davidfowl you'll need to update this after https://github.com/aspnet/Hosting/pull/992 😄